src/app/sections/ProbeDetails.svelte

```diff
@@ -194,6 +199,13 @@ h2 {
             </div>
         </div>
         {/if}
+        <div class="drawer-section">
+            <div class="helper-text--01">
+                <a href={getProbeDictURL($store.probe.apiName)} target="_blank">
```

Comment:
(1) the blue here feels very attention-drawing in the interface. I'm not sure this link needs the color, and the icon signals that it's probably a clickable link
(2) it'd be nice to get this more vertically aligned. Here's how I tend to do this (in this case, on the `a`):

```css
a {
  display: grid;
  grid-auto-flow: column;
  align-items: center;
  justify-content: end; <!-- or start, I dunno -->
  grid-column-gap: var(--space-1h);
}

```
(3) is this meant to be part of the description text? Seems like it belongs semantically near it, but that's just a hunch. It _does_ seem like it probaly belongs there vs. the `export` section.